### DISCUSSION TITLE
change texture in AR mode

### DISCUSSION THREAD

**Discussion by zjafarpishe** [google/model-viewer#4383] [Q&A] [answered]
change texture in 3d view and change texture in AR mode when use ar-modes="scene-viewer webxr"
in ios in correctly but in android dont show texture

**Comment by diegoteran** [comment, marked as answer]
Hi @zjafarpishe.
Please open a new issue and fill out the template completely, particularly the repro link, a screen recording, and any JS console errors/warnings.

**Reply by zjafarpishe** [reply]
my site http://arenvironment.arview.ir/ 
dont show texture and color in ar
